src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body,
+h2,
+p,
+ul,
+.stars {
```

Comment:
I think it's better to move it to stars.scss

Reply:
+1
but you can use just div tag here, instead of class name